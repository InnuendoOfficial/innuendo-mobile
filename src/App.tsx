// App.tsx
/**
 * This is our main React Component.
 */

import React from 'react';
import theme from './theme';
import moment from 'moment';
import 'moment/locale/fr';
import { NativeBaseProvider } from 'native-base';
import StackNavigation from './navigation/Stack';

function App() {
  moment.locale("fr")

  return (
    <NativeBaseProvider theme={theme}>
      <StackNavigation />
    </NativeBaseProvider>
  );
}

export default App