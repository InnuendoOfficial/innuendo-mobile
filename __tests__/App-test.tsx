/**
 * @format
 */

import React from 'react';
import 'react-native';
// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';
import App from '../src/App';

jest.useFakeTimers();

test('renders correctly', () => {
  const tree = renderer.create(<App />).toJSON();
  expect(tree).toMatchSnapshot();
});