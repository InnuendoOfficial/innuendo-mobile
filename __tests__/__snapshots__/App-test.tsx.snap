// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<RNCSafeAreaProvider
  onInsetsChange={[Function]}
  style={
    Array [
      Object {
        "flex": 1,
      },
      undefined,
    ]
  }
>
  <View
    style={
      Array [
        Object {
          "flex": 1,
        },
        undefined,
      ]
    }
  >
    <RNSScreenStack
      style={
        Object {
          "flex": 1,
        }
      }
    >
      <RNSScreen
        collapsable={false}
        forwardedRef={[Function]}
        nativeID="animatedComponent"
        onAppear={[Function]}
        onDisappear={[Function]}
        onDismissed={[Function]}
        onWillDisappear={[Function]}
        replaceAnimation="push"
        stackPresentation="push"
        style={
          Object {
            "bottom": 0,
            "left": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
          }
        }
      >
        <RNSScreenStackHeaderConfig
          backButtonInCustomView={false}
          backTitle=" "
          backgroundColor="#776CCB"
          color="white"
          direction="ltr"
          disableBackButtonMenu={false}
          hidden={true}
          hideBackButton={false}
          hideShadow={false}
          largeTitleHideShadow={false}
          title="Tabs"
          titleColor="white"
          topInsetEnabled={false}
          translucent={false}
        >
          <RNSScreenStackHeaderSubview
            style={
              Object {
                "alignItems": "center",
                "flexDirection": "row",
                "justifyContent": "center",
                "position": "absolute",
                "right": 0,
                "top": 0,
              }
            }
            type="center"
          >
            <Image
              resizeMode="stretch"
              source={
                Object {
                  "testUri": "../../../src/assets/logos/only_symbol_white.png",
                }
              }
              style={
                Object {
                  "height": 32,
                  "width": 32,
                }
              }
            />
          </RNSScreenStackHeaderSubview>
        </RNSScreenStackHeaderConfig>
        <View
          style={
            Array [
              Object {
                "flex": 1,
              },
              Object {
                "backgroundColor": "rgb(242, 242, 242)",
              },
              undefined,
            ]
          }
        >
          <View
            style={
              Array [
                Object {
                  "flex": 1,
                },
                undefined,
              ]
            }
          >
            <RNSScreenContainer
              style={
                Object {
                  "flex": 1,
                  "overflow": "hidden",
                }
              }
            >
              <RNSScreen
                activityState={2}
                collapsable={false}
                forwardedRef={[Function]}
                nativeID="animatedComponent"
                style={
                  Object {
                    "bottom": 0,
                    "left": 0,
                    "position": "absolute",
                    "right": 0,
                    "top": 0,
                    "zIndex": 0,
                  }
                }
              >
                <View
                  accessibilityElementsHidden={false}
                  importantForAccessibility="auto"
                  style={
                    Array [
                      Object {
                        "backgroundColor": "rgb(242, 242, 242)",
                        "flex": 1,
                      },
                      Array [
                        Object {
                          "flex": 1,
                          "flexDirection": "column-reverse",
                        },
                        undefined,
                      ],
                    ]
                  }
                >
                  <View
                    style={
                      Object {
                        "flex": 1,
                      }
                    }
                  >
                    <View
                      dataSet={Object {}}
                      style={
                        Object {
                          "alignItems": "center",
                          "flex": 1,
                          "paddingBottom": 20,
                          "paddingTop": 20,
                        }
                      }
                    >
                      <View
                        dataSet={Object {}}
                        style={
                          Object {
                            "alignItems": "center",
                            "flex": 1,
                            "width": "90%",
                          }
                        }
                      >
                        <Text
                          dataSet={Object {}}
                          style={
                            Object {
                              "backgroundColor": undefined,
                              "color": "#171717",
                              "fontFamily": "OpenSans-Bold",
                              "fontSize": null,
                              "fontStyle": undefined,
                              "fontWeight": undefined,
                              "letterSpacing": 0,
                              "lineHeight": 30,
                              "textDecorationLine": undefined,
                            }
                          }
                        >
                          Home screen
                        </Text>
                        <View
                          accessibilityRole="button"
                          accessible={true}
                          collapsable={false}
                          colorScheme="primary"
                          dataSet={Object {}}
                          elevation={3}
                          focusable={true}
                          onBlur={[Function]}
                          onClick={[Function]}
                          onFocus={[Function]}
                          onHoverIn={[Function]}
                          onHoverOut={[Function]}
                          onResponderGrant={[Function]}
                          onResponderMove={[Function]}
                          onResponderRelease={[Function]}
                          onResponderTerminate={[Function]}
                          onResponderTerminationRequest={[Function]}
                          onStartShouldSetResponder={[Function]}
                          shadowColor="black"
                          style={
                            Object {
                              "alignItems": "center",
                              "backgroundColor": "#776CCB",
                              "borderRadius": 4,
                              "flexDirection": "row",
                              "justifyContent": "center",
                              "paddingBottom": 10,
                              "paddingLeft": 12,
                              "paddingRight": 12,
                              "paddingTop": 10,
                            }
                          }
                        >
                          <View
                            dataSet={Object {}}
                            style={
                              Object {
                                "alignItems": "center",
                                "flexDirection": "row",
                              }
                            }
                            test={true}
                          >
                            <View
                              dataSet={Object {}}
                              style={Object {}}
                            >
                              <Text
                                dataSet={Object {}}
                                style={
                                  Object {
                                    "backgroundColor": undefined,
                                    "color": "#fafafa",
                                    "fontFamily": "OpenSans-Regular",
                                    "fontSize": 14,
                                    "fontStyle": undefined,
                                    "fontWeight": undefined,
                                    "letterSpacing": 0,
                                    "lineHeight": 21,
                                    "textDecorationLine": undefined,
                                  }
                                }
                              >
                                Go to edit report screen
                              </Text>
                            </View>
                          </View>
                        </View>
                      </View>
                    </View>
                  </View>
                  <View
                    onLayout={[Function]}
                    style={null}
                  >
                    <View
                      collapsable={false}
                      nativeID="animatedComponent"
                      pointerEvents="box-none"
                      style={
                        Object {
                          "bottom": 0,
                          "left": 0,
                          "position": "absolute",
                          "right": 0,
                          "top": 0,
                          "zIndex": 0,
                        }
                      }
                    >
                      <View
                        collapsable={false}
                        nativeID="animatedComponent"
                        style={
                          Object {
                            "backgroundColor": "rgb(255, 255, 255)",
                            "borderBottomColor": "#776CCB",
                            "borderBottomWidth": 2,
                            "flex": 1,
                            "shadowColor": "rgb(216, 216, 216)",
                            "shadowOffset": Object {
                              "height": 0.5,
                              "width": 0,
                            },
                            "shadowOpacity": 0.85,
                            "shadowRadius": 0,
                          }
                        }
                      />
                    </View>
                    <View
                      collapsable={false}
                      nativeID="animatedComponent"
                      pointerEvents="box-none"
                      style={
                        Object {
                          "height": 44,
                          "maxHeight": undefined,
                          "minHeight": undefined,
                          "opacity": undefined,
                          "transform": undefined,
                        }
                      }
                    >
                      <View
                        pointerEvents="none"
                        style={
                          Object {
                            "height": 0,
                          }
                        }
                      />
                      <View
                        pointerEvents="box-none"
                        style={
                          Object {
                            "alignItems": "stretch",
                            "flex": 1,
                            "flexDirection": "row",
                          }
                        }
                      >
                        <View
                          collapsable={false}
                          nativeID="animatedComponent"
                          pointerEvents="box-none"
                          style={
                            Object {
                              "alignItems": "flex-start",
                              "flexBasis": 0,
                              "flexGrow": 1,
                              "justifyContent": "center",
                              "marginStart": 0,
                            }
                          }
                        />
                        <View
                          collapsable={false}
                          nativeID="animatedComponent"
                          pointerEvents="box-none"
                          style={
                            Object {
                              "justifyContent": "center",
                              "marginHorizontal": 16,
                              "maxWidth": 718,
                            }
                          }
                        >
                          <View
                            style={
                              Object {
                                "alignItems": "center",
                                "display": "flex",
                                "flexDirection": "row",
                              }
                            }
                          >
                            <Image
                              source={
                                Object {
                                  "testUri": "../../../src/assets/logos/round.png",
                                }
                              }
                              style={
                                Object {
                                  "borderRadius": 40,
                                  "height": 32,
                                  "width": 32,
                                }
                              }
                            />
                            <Text
                              style={
                                Object {
                                  "color": "#67647D",
                                  "marginLeft": 10,
                                }
                              }
                            >
                              Innuendo
                            </Text>
                          </View>
                        </View>
                        <View
                          collapsable={false}
                          nativeID="animatedComponent"
                          pointerEvents="box-none"
                          style={
                            Object {
                              "alignItems": "flex-end",
                              "flexBasis": 0,
                              "flexGrow": 1,
                              "justifyContent": "center",
                              "marginEnd": 0,
                            }
                          }
                        />
                      </View>
                    </View>
                  </View>
                </View>
              </RNSScreen>
            </RNSScreenContainer>
            <View
              collapsable={false}
              nativeID="animatedComponent"
              onLayout={[Function]}
              pointerEvents="auto"
              style={
                Object {
                  "backgroundColor": "rgb(255, 255, 255)",
                  "borderTopColor": "#776CCB",
                  "borderTopWidth": 2,
                  "bottom": 0,
                  "elevation": 8,
                  "height": 49,
                  "left": 0,
                  "paddingBottom": 0,
                  "paddingHorizontal": 0,
                  "position": null,
                  "right": 0,
                  "transform": Array [
                    Object {
                      "translateY": 0,
                    },
                  ],
                }
              }
            >
              <View
                pointerEvents="none"
                style={
                  Object {
                    "bottom": 0,
                    "left": 0,
                    "position": "absolute",
                    "right": 0,
                    "top": 0,
                  }
                }
              />
              <View
                accessibilityRole="tablist"
                style={
                  Object {
                    "flex": 1,
                    "flexDirection": "row",
                  }
                }
              >
                <View
                  accessibilityLabel="Home, tab, 1 of 2"
                  accessibilityRole="button"
                  accessibilityState={
                    Object {
                      "selected": true,
                    }
                  }
                  accessibilityStates={
                    Array [
                      "selected",
                    ]
                  }
                  accessible={true}
                  collapsable={false}
                  focusable={true}
                  onBlur={[Function]}
                  onClick={[Function]}
                  onFocus={[Function]}
                  onResponderGrant={[Function]}
                  onResponderMove={[Function]}
                  onResponderRelease={[Function]}
                  onResponderTerminate={[Function]}
                  onResponderTerminationRequest={[Function]}
                  onStartShouldSetResponder={[Function]}
                  style={
                    Array [
                      Object {
                        "alignItems": "center",
                        "flex": 1,
                      },
                      Object {
                        "backgroundColor": "transparent",
                      },
                      Object {
                        "flexDirection": "column",
                        "justifyContent": "flex-end",
                      },
                      undefined,
                    ]
                  }
                >
                  <View
                    style={
                      Array [
                        Object {
                          "flex": 1,
                        },
                        undefined,
                      ]
                    }
                  >
                    <View
                      style={
                        Array [
                          Object {
                            "alignItems": "center",
                            "alignSelf": "center",
                            "height": "100%",
                            "justifyContent": "center",
                            "minWidth": 25,
                            "position": "absolute",
                            "width": "100%",
                          },
                          Object {
                            "opacity": 1,
                          },
                        ]
                      }
                    >
                      <Text
                        allowFontScaling={false}
                        style={
                          Array [
                            Object {
                              "color": "#776CCB",
                              "fontSize": 25,
                            },
                            undefined,
                            Object {
                              "fontFamily": "Ionicons",
                              "fontStyle": "normal",
                              "fontWeight": "normal",
                            },
                            Object {},
                          ]
                        }
                      >
                        
                      </Text>
                    </View>
                    <View
                      style={
                        Array [
                          Object {
                            "alignItems": "center",
                            "alignSelf": "center",
                            "height": "100%",
                            "justifyContent": "center",
                            "minWidth": 25,
                            "position": "absolute",
                            "width": "100%",
                          },
                          Object {
                            "opacity": 0,
                          },
                        ]
                      }
                    >
                      <Text
                        allowFontScaling={false}
                        style={
                          Array [
                            Object {
                              "color": "#AFACC6",
                              "fontSize": 25,
                            },
                            undefined,
                            Object {
                              "fontFamily": "Ionicons",
                              "fontStyle": "normal",
                              "fontWeight": "normal",
                            },
                            Object {},
                          ]
                        }
                      >
                        
                      </Text>
                    </View>
                  </View>
                </View>
                <View
                  accessibilityLabel="Calendar, tab, 2 of 2"
                  accessibilityRole="button"
                  accessibilityState={
                    Object {
                      "selected": false,
                    }
                  }
                  accessibilityStates={Array []}
                  accessible={true}
                  collapsable={false}
                  focusable={true}
                  onBlur={[Function]}
                  onClick={[Function]}
                  onFocus={[Function]}
                  onResponderGrant={[Function]}
                  onResponderMove={[Function]}
                  onResponderRelease={[Function]}
                  onResponderTerminate={[Function]}
                  onResponderTerminationRequest={[Function]}
                  onStartShouldSetResponder={[Function]}
                  style={
                    Array [
                      Object {
                        "alignItems": "center",
                        "flex": 1,
                      },
                      Object {
                        "backgroundColor": "transparent",
                      },
                      Object {
                        "flexDirection": "column",
                        "justifyContent": "flex-end",
                      },
                      undefined,
                    ]
                  }
                >
                  <View
                    style={
                      Array [
                        Object {
                          "flex": 1,
                        },
                        undefined,
                      ]
                    }
                  >
                    <View
                      style={
                        Array [
                          Object {
                            "alignItems": "center",
                            "alignSelf": "center",
                            "height": "100%",
                            "justifyContent": "center",
                            "minWidth": 25,
                            "position": "absolute",
                            "width": "100%",
                          },
                          Object {
                            "opacity": 0,
                          },
                        ]
                      }
                    >
                      <Text
                        allowFontScaling={false}
                        style={
                          Array [
                            Object {
                              "color": "#776CCB",
                              "fontSize": 25,
                            },
                            undefined,
                            Object {
                              "fontFamily": "Ionicons",
                              "fontStyle": "normal",
                              "fontWeight": "normal",
                            },
                            Object {},
                          ]
                        }
                      >
                        
                      </Text>
                    </View>
                    <View
                      style={
                        Array [
                          Object {
                            "alignItems": "center",
                            "alignSelf": "center",
                            "height": "100%",
                            "justifyContent": "center",
                            "minWidth": 25,
                            "position": "absolute",
                            "width": "100%",
                          },
                          Object {
                            "opacity": 1,
                          },
                        ]
                      }
                    >
                      <Text
                        allowFontScaling={false}
                        style={
                          Array [
                            Object {
                              "color": "#AFACC6",
                              "fontSize": 25,
                            },
                            undefined,
                            Object {
                              "fontFamily": "Ionicons",
                              "fontStyle": "normal",
                              "fontWeight": "normal",
                            },
                            Object {},
                          ]
                        }
                      >
                        
                      </Text>
                    </View>
                  </View>
                </View>
              </View>
            </View>
          </View>
        </View>
      </RNSScreen>
    </RNSScreenStack>
  </View>
</RNCSafeAreaProvider>
`;
